org.splash.Splash.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Emmanuel Durand -->
+<component type="desktop-application">
```

Comment:
Not a blocker: this should be upstream as it's not a Flathub specific thing.

Reply:
Right, I moved it!